spec/jobs/whitehall_document_import_job_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+RSpec.describe WhitehallDocumentImportJob do
+  include ActiveJob::TestHelper
+
+  let(:whitehall_migration) {
+    build(:whitehall_migration)
+  }
```

Comment:
Minor: our convention dictates do and end for multi line blocks. This should probably be a single line block.